index.html

```diff
@@ -10,46 +10,236 @@
 </head>
 
 <body>
-  
   <header>
-  
     <a href="#about" class="logo">Joy</a>
-    
     <nav title="navigation-bar">
       <ul class="ul-nav" title="navigation-list">
         <li><a class="navigationList" href="#portfolio">Portfolio</a></li>
         <li><a class="navigationList" href="#about">About</a></li>
         <li><a class="navigationList" href="#contactme">Contact</a></li>
       </ul>
-    
-      <div class="nav-button2">
+       <div class="nav-button2">
         <span class="bar"></span>
         <span class="bar"></span>
         <span class="bar"></span>
-      </div>
+       </div>
     </nav>
   </header>
 
 <!--Page 1-->
 <section class="mobilemain" title="Joy's-Portfolio-Home-page">
  <article class="headline">
-   <h1 id="heading">I’m Joy<br>
-     Glad to see you!</h1>
+   <h1 id="heading">I’m Joy<br>Glad to see you!</h1>
+     
+     <p id="paragraph1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+     
+     <div class="smedia" title="social-media-accounts">
+      <a class="LetsConnect" href="#">LET’S CONNECT </a>
+      
+      <ul class="topicons">
+        <li><a href="#"> <img src="sources/Vector4.png" alt="twitter"></a></li>    
+        <li><a href="#"> <img src="sources/linkedin.png" alt="linkedin"></a></li>     
+        <li><a href="#"> <img src="sources/Vector5.png" alt="medium"></a></li>        
+        <li><a href="#"> <img src="sources/Vector1.png" alt="github"></a></li>        
+        <li><a href="#"> <img src="sources/Vector3.png" alt="chill"></a></li>   
+      </ul>
+   
+     </div>
+    </article>
+
+
+<!--page 2-->
+<section class="works" id="portfolio" title="works-card-1">
+  
+  <div class="card1 cards">
+    <img class="avator3" id="avator" src="sources/snap.png" alt="Avatar">
+    <div class="container">
+      <h2><b>Tonic</b></h2>
+
+     <ul class="tonic">
```

Comment:
The project card is not fully centered as per Figma Design.
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219083621-84daa019-0892-4c83-b96b-9dd5c6653c5f.png)  | ![image](https://user-images.githubusercontent.com/61361037/219083813-dcd4244c-c02e-4f1a-964a-27b2d4f5d029.png)  |